fieldservice_stock/__manifest__.py

```diff
@@ -12,6 +12,7 @@
     'website': 'https://github.com/OCA/field-service',
     'depends': [
         'fieldservice',
+        'stock_request_direction',
```

Comment:
```suggestion
        'fieldservice_geoengine',
        'stock_request_direction',
```

Reply:
It will not solve the Travis error. See https://github.com/OCA/geospatial/issues/206